skillserver/echo.go

```diff
@@ -57,6 +57,27 @@ func (this *EchoRequest) AllSlots() map[string]EchoSlot {
 	return this.Request.Intent.Slots
 }
 
+// Locale returns the locale specified in the request.
+//
+// If `locales` is provided, the first matching locale will be returned,
+// falling back to the first provided value if none match the request.
+//
+// (Matching is performed via a strict string comparison. For more
+// advanced matching, you may want to use the "golang.org/x/text/language"
+// library instead.)
+func (this *EchoRequest) Locale(locales ...string) string {
```

Comment:
what is the reason for providing a list of locales? i'm not sure why the client wouldn't just request the locale and then check it against some knowns values or a default if the locale is empty.

Reply:
I agree with @rking788 here. I think a Locale accessor could be useful but I don't understand why we need the fancy matching feature.

Reply:
I figured a "fetch locale and compare against list of supported locales (with fallback)" use case would be common enough to make `.Locale("en-US", "fr-FR", "de-DE")` worthwhile, based on my past experience.

However, given the triviality of the code, and the fact that (unlike, e.g. HTTP requests) Alexa requests are restricted to those specified within the skill configuration, I agree this is probably unnecessary!

I'll put forward an updated PR shortly.